config/initializers/redis.rb

```diff
@@ -0,0 +1,13 @@
+$redis = Redis.new(:host => 'localhost', :port => 3000)
+
+if defined?(Unicorn)
```

Comment:
You should have added at least some info about these changes in your PR message.